api/controllers/performances-controller.js

```diff
@@ -18,17 +18,30 @@ class PerformancesController {
     });
   }
 
-  static index(req, res) {
+  static index(req, res, next) {
     Performance.findAll('MMMM Do, h:mm:ss a')
     .then((performances) => {
-      res.render('performances/index', { user: req.user, performances: performances.map((performance) => performance.toJSON()) });
+      res.locals.jsonResp = { performances };
+      next();
     })
     .catch((err) => {
       logger.errorLog('Performances.index', err);
       res.render('static-pages/error', { user: req.user });
     });
   }
 
+  static update({ body }, res, next) {
+    Performance.update(body)
```

Comment:
This needs data validation, but that's something that's in the pipeline. Not in the scope of this PR